Gopkg.toml

```diff
@@ -68,3 +68,19 @@
 [[constraint]]
   branch = "master"
   name = "github.com/lxn/walk"
+
+[[constraint]]
+  name = "github.com/jaypipes/ghw"
+  version = "0.3.0"
+
+[[constraint]]
+  branch = "master"
```

Comment:
Shouldn't we ping to a version or commit instead simply `master`?

Reply:
done